packages/babel-plugin-transform-typescript/test/fixtures/regression/15768/input.ts

```diff
@@ -0,0 +1,11 @@
+const v = 42;
+const v2 = Infinity;
+var Infinity = 1; // Known inconsistencies
```

Comment:
Note that in scripts this doesn't actually change the value of `Infinity`. However, there is another case in which it does and TS is not behaving as I would expect: https://github.com/microsoft/TypeScript/issues/54981